recipes/spectrapepper/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "spectrapepper" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7bf87dbc5ada99d3ba4d1b12d99f052d8154d39c6614698d09ea80014cca4f97
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib
```

Comment:
Unless you need a pyqt gui, best to depend on `matplotlib-base`

Reply:
@dopplershift Thank you for your revision. Indeed it does not need a pyqt gui, so I have changed it to `matplotlib-base`.